tests/fake_auth_session.py

```diff
@@ -20,16 +24,22 @@ def test_reset(cls):
         cls.test_delete_called = False
 
     @classmethod
-    def get(cls, session_id):
-        from tests.fake_user import FakeUser
-        identity = FakeUser(username='teste@fakeauthsession.com', password='1234567a')
-        identity.uuid = session_id
-        return cls(identity, True, 600, arrow.utcnow().datetime)
+    def get_identity(cls, username):
```

Comment:
Ficou estranho um get_identity retornar um AuthSession

```suggestion
    def get_by_username(cls, username):
```

```suggestion
    def get_by_identity(cls, username):
```

Reply:
mas ele não vai retornar um identity + permissions?

Reply:
Alterei na interface que ele recebe um username e não mais um session_id